scripts/lldb_pretty_printers/data_path_pretty_printer.py

```diff
@@ -0,0 +1,36 @@
+import lldb
+
+def data_path_summary(valobj, internal_dict):
+    """
+    Return a string summary for nx::core::DataPath.
+    We'll display the contents of m_Path (std::vector<std::string>)
+    as a list of unquoted string values.
+    """
+    path_valobj = valobj.GetChildMemberWithName('m_Path')
```

Comment:
We might consider a different approach since `m_Path` is a private variable and an implementation detail so if it changes then this breaks. It is possible to call expressions in lldb like so: `valobj.EvaluateExpression('toString()')`. I think it's worth looking into at least.

Reply:
Unfortunately that does not work on macOS 14. Will stay with the implementation details version.
![Screenshot 2025-02-19 at 09 43 23](https://github.com/user-attachments/assets/965379ed-3cf9-48c2-8169-0ee9d620b32e)